core/util/runPuppet.js

```diff
@@ -211,12 +227,22 @@ async function processScenarioView (scenario, variantOrScenarioLabelSafe, scenar
       config,
       result.backstopSelectorsExp,
       result.backstopSelectorsExpMap
-    ));
+    )
+  } catch (e) {
+    error = e;
+  }
+  
+  return new Promise((resolve, reject) => {
+    if (compareConfig) {
+      resolve(compareConfig);
+    } else {
+      resolve(new BackstopException('Puppeteer error', scenario, viewport, error));
```

Comment:
Shouldn't this be `reject` instead of `resolve`? Because now, the promise resolves with an error, which should be a reject or?

Reply:
In this case we resolve either way.   The intention is to pass the error up as a result.